app/views/users/phone_setup/index.html.erb

```diff
@@ -37,20 +44,15 @@
       ).with_content(t('forms.buttons.send_one_time_code')) %>
 <% end %>
 
-<p class="margin-top-5">
-  <%= t('two_factor_authentication.phone_fee_disclosure') %>
-  <% if IdentityConfig.store.voip_block %>
-    <%= t('two_factor_authentication.two_factor_choice_options.phone_info_no_voip') %>
-  <% end %>
-</p>
-<% unless FeatureManagement.phone_recaptcha_enabled? %>
-  <%= link_to t('two_factor_authentication.mobile_terms_of_service'),
-              MarketingSite.messaging_practices_url,
-              class: 'display-block margin-bottom-4' %>
-<% end %>
```

Comment:
Should this be preserved?

Reply:
The link was not a part of the page update because the link is now incorporated into the full reCAPTCHA disclosure

Reply:
> The link was not a part of the page update because the link is now incorporated into the full reCAPTCHA disclosure

But the reCAPTCHA disclosure is only shown when the feature is enabled, so we'll still want this text if that feature is disabled?

At some point, we might want to consider removing the feature flag behavior to avoid having to juggle all the moving pieces.